DOCS-2305: Email corrections (#1306)
MultiSafepay/docs: content/payment-methods/wallet/applepay/activate-apple-pay.md

When approved by our Risk Team, Visa, Mastercard and Apple Pay will be enabled in your MultiSafepay Control as payment methods.

> _If credit cards are already enabled as a payment method, email <integration@multisafepay.com> and we will activate Apple Pay for you._

Reviewer: ```suggestion
_If credit cards are already enabled as a payment method, to activate Apply Pay  email the Integration Team at <integration@multisafepay.com> 
```
